index.web.js

```diff
@@ -8,27 +8,50 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { Router, Route, browserHistory } from 'react-router';
 import { createStore, applyMiddleware, combineReducers } from 'redux';
-import { syncHistoryWithStore, routerReducer } from 'react-router-redux';
 import { Provider } from 'react-redux';
 import Thunk from 'redux-thunk';
+import {
+    syncHistoryWithStore,
+    routerReducer,
+    routerMiddleware,
+    push
+} from 'react-router-redux';
```

Comment:
I'd rather we have imports alphabetically sorted by URL, location or whatever that thing after the 'from' is called. That makes it so much easier for me to figure out where I need to add new imports. But that may be just me so I'm open to discussion.

react-dom
react-redux
react-router
react-router-redux
redux
redux-thunk